TF_Tooling_Web/src/Connection/ServiceConnection.ts

```diff
@@ -134,6 +134,7 @@ export class ServiceConnection {
 
             case ActionCode.CLOSE_TO_SWIPE:
                 CursorManager.instance.cursor.ShowCloseToSwipe();
+                CursorManager.instance.cursor.SetSwipeDirection(looseData.content);
```

Comment:
Do we need SetSwipeDirection or is passing the content to ShowCloseToSwipe enough?